agency_client/src/utils/validation.rs

```diff
@@ -26,14 +27,15 @@ pub fn validate_did(did: &str) -> AgencyClientResult<String> {
                 return Err(AgencyClientError::from_msg(AgencyClientErrorKind::InvalidDid, "Invalid DID length"));
```

Comment:
I know this is unrelated to your PR, but logging
```
warn!("ok(_)");
```
on line above is quite uninformative